pkg/apk/version.go

```diff
@@ -370,36 +370,50 @@ func (v versionDependency) satisfies(actualVersion, requiredVersion packageVersi
 	}
 }
 
-func resolvePackageNameVersionPin(pkgName string) (name, version string, dep versionDependency, pin string) {
+type pinStuff struct {
+	name    string
+	version string
+	dep     versionDependency
+	pin     string
+}
+
+func resolvePackageNameVersionPin(pkgName string) pinStuff {
 	parts := packageNameRegex.FindAllStringSubmatch(pkgName, -1)
 	if len(parts) == 0 || len(parts[0]) < 2 {
-		return pkgName, version, versionNone, pin
+		return pinStuff{
+			name: pkgName,
+			dep:  versionNone,
+		}
 	}
 	// layout: [full match, name, =version, =|>|<, version, @pin, pin]
```

Comment:
I think this comment is out of date?

Reply:
I think it's still right? It's describing `parts`, which is unchanged.